src/main/java/nl/tudelft/context/controller/MenuController.java

```diff
@@ -35,79 +32,114 @@
-     * Set up file menu.
+     * The function that will create a menu with an undefined number of menuItems.
+     *
+     * @param title     Title of the menu.
+     * @param menuItems The menu items that will be added to this menu.
+     * @return Menu with the menuItems attached.
      */
-    public void initFileMenu() {
+    private Menu createMenu(final String title, final MenuItem... menuItems) {
 
-        final Menu fileMenu = new Menu("_File");
-
-        MenuItem load = new MenuItem("Load Workspace...");
-        load.setAccelerator(new KeyCodeCombination(KeyCode.O, KeyCombination.SHORTCUT_DOWN));
-        DirectoryChooser directoryChooser = new DirectoryChooser();
-        directoryChooser.setTitle("Select Workspace Folder");
-        load.setOnAction(event -> Workspace.chooseWorkspace(mainController));
-
-        MenuItem exit = new MenuItem("Exit");
-        exit.setOnAction(event -> mainController.exitProgram());
-
-        fileMenu.getItems().addAll(load, exit);
-        menuBar.getMenus().add(fileMenu);
+        final Menu menu = new Menu(title);
+        menu.getItems().addAll(menuItems);
+        return menu;
 
     }
 
     /**
-     * Set up navigate menu.
+     * The function that returns a menuItem with a title, shortcut and event attached.
+     *
+     * @param title   The title of the menuItem.
+     * @param keyComb The shortcut to this menuItem.
+     * @param event   Event that the item will use.
+     * @return The menuItem with the parameters attached.
      */
-    public void initNavigateMenu() {
+    private MenuItem createMenuItem(final String title,
+                                    final KeyCombination keyComb,
+                                    final EventHandler<ActionEvent> event) {
 
-        final Menu navigateMenu = new Menu("_Navigate");
+        final MenuItem res = new MenuItem(title);
+        res.setAccelerator(keyComb);
+        res.setOnAction(event);
 
-        final MenuItem previous = new MenuItem("Previous");
-        previous.setAccelerator(new KeyCodeCombination(KeyCode.ESCAPE));
-        previous.setOnAction(event -> mainController.previousView());
+        return res;
 
-        loadGenomeGraph = new MenuItem("Load Genome Graph");
-        loadGenomeGraph.setAccelerator(new KeyCodeCombination(KeyCode.L, KeyCombination.SHORTCUT_DOWN));
-        loadGenomeGraph.setDisable(true);
+    }
 
-        final MenuItem showNewick = new MenuItem("Show Phylogenetic tree");
-        showNewick.setAccelerator(new KeyCodeCombination(KeyCode.T, KeyCombination.SHORTCUT_DOWN));
-        showNewick.setOnAction(event -> mainController.toggleNewick());
+    /**
+     * The function that returns a menuItem without a shortcut.
+     *
+     * @param title Title of the menuItem.
+     * @param event Event that the item will use.
+     * @return The menu item with the title and the event attached.
+     */
+    private MenuItem createMenuItem(final String title, final EventHandler<ActionEvent> event) {
```

Comment:
This is the same as `createMenuItem(title, null, event)`.

When you choose to overload functions make sure the order of the parameters is the same. Better in this case would be:

* `createMenuItem(title, event)`
* `createMenuItem(title, event, keyComb)`

Also, DRY, you can use `createMenuItem(title, event)` in `createMenuItem(title, event, keyComb)`.